core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -292,6 +324,23 @@ public MethodMetrics(ResourceMethod method, PerformanceMetric annotation, Metric
       this.errorSensor.add(metricName, new CumulativeCount());
     }
 
+    private void setResponseLatencySloSlaSensor(ResourceMethod method,
+        PerformanceMetric annotation, Metrics metrics, Map<String, String> requestTags,
+        String metricGrpName, Map<String, String> allTags) {
+      for (int i = 0; i < LATENCY_SLO_SLA_SENSOR_NAMES.length; i++) {
+        final Sensor sensor = metrics.sensor(
+            getName(method, annotation, LATENCY_SLO_SLA_SENSOR_NAMES[i], requestTags),
+            null, SENSOR_EXPIRY_SECONDS, Sensor.RecordingLevel.INFO, (Sensor[]) null);
+        MetricName metricName = new MetricName(
+            getName(method, annotation, LATENCY_SLO_SLA_SENSOR_NAMES[i] + "-total"), metricGrpName,
+            (i % 2 == 0 ? "Below" : "Above")
+                + " latency SLA request count, using a cumulative counter",
```

Comment:
This seems like it could be improved. The array of names is a bit awkward in practice, as you can see when the "latency SLA" string is being used even for SLO-related metrics. I would restructure this code and remove the array of names.